### DISCUSSION TITLE
I had a good idea but I didn't manage to correctly finish it... Need help!

### DISCUSSION THREAD

**Discussion by ricky732** [theOehrly/Fast-F1#87] [Ideas]
Hiii guys!! Especially @theOehrly.. I had a good idea, extracting the acceleration curve from the data, to see traction and braking power. I studied the method, converted the speed in m/s, but to finish it, I need to convert the timedelta dataframe to intervals in seconds and than creating a new array / dataframe. Every element of this new dataframe / array I want to be delta speed / delta time, where the delta is referred to every 2 successive data-refresh. Any ideas about how to finish this project? 🙂

**Comment by ricky732** [comment]
Just thought this... If the refresh of the data is constant (every tot milliseconds) and someone can tell me what's that interval of refreshing the data, it would be much easier. No need to convert timedelta, just to create a new dataframe with speed deltas (absolutely no idea about how to do it 😅😢)

**Comment by siddharth-tewari** [comment]
@ricky732 I have a little bit of idea so trying to make.

**Comment by siddharth-tewari** [comment]
@ricky732 Oh wrong issue. I meant for your speed-trap idea.

**Comment by marinik98** [comment]
Polling rate is 5Hz, just way to small to make any accurate consideration of less than a couple of seconds events.
